mitosheet/src/components/taskpanes/ExportToFile/ExportToFileTaskpane.tsx

```diff
@@ -0,0 +1,161 @@
+                            File Name
+                        </p>
+                    </Col>
+                    <Col>
+                        <Input
+                            autoFocus
+                            width='medium'
+                            value={'' + params.file_name}
+                            onChange={(e) => {
+                                const newValue = e.target.value;
+                                
+                                setParams(prevParams => {
+                                    return {
+                                        ...prevParams,
+                                        file_name: newValue
+                                    }
+                                })
+                            }}
+                        />
+                    </Col>
+                </Row>
+                <Row justify='space-between' align='center'>
+                    <Col>
+                        <p className='text-header-3'>
+                            File Type
+                        </p>
+                    </Col>
+                    <Col>
+                        <Select 
+                            width="medium"
+                            value={params.type}
+                            onChange={(newType) => {
+                                setParams(prevParams => {
+                                    return {
+                                        ...prevParams,
+                                        type: newType as 'csv' | 'excel'
+                                    }
+                                })
+                            }}
+                        >
+                            <DropdownItem title="CSV" id='csv' subtext="Each exported dataframe will be exported as a seperate CSV file."/>
+                            <DropdownItem title="Excel" id='excel' subtext="Each exported dataframe will be exported as a seperate sheet."/>
+                        </Select>
+                    </Col>
+                </Row>
+                <Row>
+                    <Col>
+                        <p className="text-header-3">Dataframes to Export</p>
+                    </Col>
+                </Row>
+                <DataframeMultiSelect
+                    sheetDataArray={props.sheetDataArray}
+                    selectedSheetIndexes={params.sheet_indexes}
+                    setUIState={props.setUIState}
+                    onChange={(newSelectedSheetIndexes) => {
+                        setParams(prevParams => {
+                            return {
+                                ...prevParams,
+                                sheet_indexes: newSelectedSheetIndexes
+                            }
+                        })
+                    }}
+                />
+            </DefaultTaskpaneBody>
+            <DefaultTaskpaneFooter>
+                {editApplied && <p className='text-body-3'>Files and code written.</p>}
```

Comment:
Can we update the `Files and code written.` result subtext to just say `Generated export code` 

And can we make it subtext instead of text-body-3

Reply:
I think it's good to say the files as well, but I updated the language to be more clear!